inst/include/Model.h

```diff
@@ -514,21 +514,60 @@ class Model : public ShallowCopyable{
 			offsets[k]->vDyadUpdate(*net, from, to);
 		}
 	}
-
+    
+    void dyadUpdateR(int from, int to){
+        if(from > net->size() || to > net->size())
+            ::Rf_error("one of the vertex indices in the dyad update is bigger than the size of the network");
+        if(from <1 || to <1)
+            ::Rf_error("one of the vertex indices in the dyad update is less than or equal to 0");
+        this->dyadUpdate((from-1),(to-1));
+    }
 
 	void discreteVertexUpdate(int vertex, int variable, int newValue){
 		for(int k=0;k<stats.size();k++)
 			stats[k]->vDiscreteVertexUpdate(*net,vertex, variable, newValue);
 		for(int k=0;k<offsets.size();k++)
 			offsets[k]->vDiscreteVertexUpdate(*net,vertex, variable, newValue);
 	}
+    
+    void discreteVertexUpdateR(int vertex, std::string varName, int newValue){
```

Comment:
I thought these changes were already merged in.

Reply:
I don't know. It is possible that it would look better if you were doing the pull requests from branches rather than a forked repo.